mcs/nunit24/NUnitCore/core/BabysitterSupport.cs

```diff
@@ -0,0 +1,104 @@
+// Added to NUnit by Andi McClure to add special support for the test babysitter script.
```

Comment:
You can use our standard file header like http://www.mono-project.com/community/contributing/coding-guidelines/#file-headers